book/coming_from_cmd.md

```diff
@@ -1,6 +1,6 @@
 # Coming from CMD.EXE
 
-This table was last updated for Nu 0.67.0.
+This table was last updated for Nu 0.99.1.
```

Comment:
I did not go over the command table. I was unsure whether it should remain the previous, be updated to the new, or split across concerns.

I chose to update it for simplicity, thinking issues can still be resolved either way when they come up/get reported.